pages/exercise/index.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * @defgroup pages_announcement Announcements page
+ */
+
+/**
+ * @file pages/exercise/index.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2003-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @ingroup pages_exercise
+ *
+ * @brief Handle requests for about the journal functions.
+ *
+ */
+
+use APP\pages\exercise\AnnouncementHandler;
```

Comment:
There's a mismatch here between `exercise` (the page name, which becomes part of the URL) and announcements (the handler name, the `@defgroup`, etc). If this were going to be merged, we would reconcile those for consistency. But as this is just an exercise, no need to do that here, I think.